src/blocksmith/plugins/FloatingTextFormatToolbarPlugin.js

```diff
@@ -202,14 +208,24 @@ function FloatingTextFormatToolbar({
             <Icon imgSrc="link" />
           </button>
         ) : (
-          <button
-            onClick={handleFormat(TOGGLE_LINK_COMMAND)}
-            className="toolbar-item active"
-            aria-label="Remove link"
-            type="button"
-          >
-            <Icon imgSrc="unlink" />
-          </button>
+          <>
+            <button
+              onClick={handleInsertLink}
+              className="toolbar-item active"
+              aria-label="Edit link"
+              type="button"
+            >
+              <Icon imgSrc="link" />
+            </button>
```

Comment:
@joshlevy this seems almost identical to the previous button except for the class and label